packages/oauth/src/install-provider.ts

```diff
@@ -9,58 +9,91 @@ import {
   InstallerInitializationError,
   UnknownError,
   MissingStateError,
+  InvalidStateError,
   MissingCodeError,
   GenerateInstallUrlError,
   AuthorizationError,
   CodedError,
   ErrorCode,
 } from './errors';
-import { Installation, OrgInstallation } from './installation';
+import { Installation } from './installation';
 import { InstallationQuery } from './installation-query';
 import { InstallURLOptions } from './install-url-options';
 import { InstallProviderOptions } from './install-provider-options';
 import { Logger, LogLevel, getLogger } from './logger';
 import { ClearStateStore, StateStore } from './state-stores';
 import { InstallationStore, MemoryInstallationStore } from './stores';
+import defaultRenderHtmlForInstallPath from './default-render-html-for-install-path';
 
 /**
- * InstallProvider Class.
- * @param clientId - Your apps client ID
- * @param clientSecret - Your apps client Secret
- * @param stateSecret - Used to sign and verify the generated state when using the built-in `stateStore`
- * @param stateStore - Replacement function for the built-in `stateStore`
- * @param stateVerification - Pass in false to disable state parameter verification
- * @param installationStore - Interface to store and retrieve installation data from the database
- * @param authVersion - Can be either `v1` or `v2`. Determines which slack Oauth URL and method to use
- * @param logger - Pass in your own Logger if you don't want to use the built-in one
- * @param logLevel - Pass in the log level you want (ERROR, WARN, INFO, DEBUG). Default is INFO
+ * InstallProvider Class. Refer to InsallProviderOptions interface for the details of constructor arguments.
  */
 export class InstallProvider {
+  // Manages state parameter values in the OAuth flow
   public stateStore?: StateStore;
 
+  // Manages installation data, which can be called by both the OAuth flow and authorize() in event handling
   public installationStore: InstallationStore;
 
+  // Client ID, which can be found at https://api.slack.com/apps
```

Comment:
Same comment as above, perhaps we can be more specific as to where to find these values on the App Config page. Granted, it may be too specific. I do not have a strong feeling about this comment.